content/docs/examples/_index.md

```diff
@@ -30,6 +30,7 @@ If you find a good example, add it via a pull request, please! Curious how? Foll
 
 ## Websites and platforms
 - [Website of this course](https://dprep.hannesdatta.com)
+- [Website of complementary course](http://odcm.hannesdatta.com)
```

Comment:
please change to: Website of complementary course to collect web data using scraping and APIs